app/services/export_files/packager.rb

```diff
@@ -51,5 +56,25 @@ def permitted?(obj)
       ability.can?(:read, obj)
     end
 
+    # Logic adapted from Hyrax::MemberPresenterFactory#ordered_ids
+    def ordered_member_ids(work_doc)
+      query = "proxy_in_ssi:#{work_doc.id}"
+      results = ActiveFedora::SolrService.query(query, rows: 10_000, fl: 'ordered_targets_ssim')
+      results.flat_map { |x| x.fetch("ordered_targets_ssim", []) }
+    end
+
+    # Logic adapted from Hyrax::MemberPresenterFactory#file_set_ids
+    def file_set_ids(work_doc)
+      query = "{!field f=has_model_ssim}FileSet"
+      filter = "{!join from=ordered_targets_ssim to=id}id:\"#{work_doc.id}/list_source\""
+      results = ActiveFedora::SolrService.query(query, rows: 10_000, fl: ActiveFedora.id_field, fq: filter)
+      results.flat_map { |x| x.fetch(ActiveFedora.id_field, []) }
+    end
+
+    # Logic suggested by Hyrax::MemberPresenterFactory#work_presenters
+    def ordered_work_ids(work_doc)
+      ordered_member_ids(work_doc) - file_set_ids(work_doc)
+    end
+
   end
```

Comment:
Would it make a difference to pass parm as work_doc.ids between the above methods, rather than object work_doc?